docs/Contributing/architecture/software/software-installation.md

```diff
@@ -0,0 +1,103 @@
+# Software Installation Architecture
```

Comment:
This article needs to be rewritten. It divides up responsibilities incorrectly, and when it gets into details on what does what, it's wrong. Would be better to merge this with a TODO, with a ticket filed (happy to take said ticket) to write something up that is an add-on to the existing software installation article.

Reply:
Feel free to suggest changes! That's my thought, that the stuff that's wrong we just replace with TODOs.

Reply:
I'm emptying this of everything except headers.